### DISCUSSION TITLE
After fpv.py is executed, it shows Could not connect to Crazyflie

### DISCUSSION THREAD

**Discussion by jackyYu416** [bitcraze/discussions#474] [Q&A] [answered]
Hi everyone, I am showing Could not connect to Crazyflie error after executing fpv.py, it has never happened before and it makes me overwhelmed
![image](https://user-images.githubusercontent.com/117829285/207863151-5dbad034-a1a9-4e72-a9fe-e07cf8f88458.png)

**Comment by krichardsson** [comment]
Please rephrase your question using English

**Reply by jackyYu416** [reply]
Sorry for not switching to English, thanks for the reminder

**Reply by evoggy** [reply]
Could it be that the hostname is incorrect? Are you using the AI deck as access point or is it connecting to an already existing wifi network?

**Reply by jackyYu416** [reply]
I use 'tcp://aideck-AABBCC.local:5000' AABBCC is aideck's ip, and connect to wifi streaming example. However my laptop can't connect, another public computer can but only loses connection after three seconds of takeoff.

**Comment by jackyYu416** [comment, marked as answer]
Oh, I found out what the problem was, I didn't notice when I updated the December firmware that I connected both the flow deck and the aideck! I've fixed the problem，thanks guys
